frontend/src/index.js

```diff
@@ -30,8 +32,7 @@ ReactDOM.render(
         <Link to={paths.home} className={styles.header}>
           OASIS
         </Link>
-        <Map></Map>
-        <main className={styles.root}>
+        <Wrapper draggableMapRoutes={[paths.dashboard, "/mymap"]}>
```

Comment:
Are we leaving the `/mymap` route? if we are, we should make it a path. I thought it was just a test sandbox.